### PR TITLE
Adds a FAQ item about cleaning up images

### LINKED ISSUE #355: FAQ Item: "How do I purge or prune old containerd images from Bottlerocket?"
**Path to page with content (_do not_ include domain name):**

`/en/faq/`

**What content is incorrect?**

There should be a question about purging container images. Multiple people have asked this question:

- https://github.com/bottlerocket-os/bottlerocket/issues/3568
- https://github.com/bottlerocket-os/bottlerocket/discussions/2131

**What should the content say or communicate?**

It should show how to manage this using:
- [ctr/address](https://github.com/bottlerocket-os/bottlerocket/issues/3568#issuecomment-1792880120)
- [imageGC* settings](https://github.com/bottlerocket-os/bottlerocket/discussions/2131#discussioncomment-2777730)

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Closes #355

**Description of changes:**
Adds FAQ item to tell uses how to clean up images. 

It also includes a small change the `ver-ref` shortcode that allows for hash links.

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
++++
+question = "How do I clean up old container images from Bottlerocket?"
+group = "Mangagement"
```
Comment (bcressey) [inline content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown:3]: Typo

File: content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+On ECS variants, the [ECS Agent](https://github.com/aws/amazon-ecs-agent) manages the container image clean up, the settings `{{< ver-ref project="os" page="/api/settings/ecs/" hash="#image-cleanup-enabled" >}}settings.ecs.image-cleanup-enabled{{</ ver-ref>}}`, `{{< ver-ref project="os" page="/api/settings/ecs/" hash="#image-cleanup-age" >}}settings.ecs.image-cleanup-age{{</ ver-ref>}}`, `{{< ver-ref project="os" page="/api/settings/ecs/" hash="#image-cleanup-delete-per-cycle" >}}settings.ecs.image-cleanup-delete-per-cycle{{</ ver-ref>}}`, and
+`{{< ver-ref project="os" page="/api/settings/ecs/" hash="#image-cleanup-wait" >}}settings.ecs.image-cleanup-wait{{</ ver-ref>}}` allow you to control how this clean up occurs.
+
+If you want to manually clean up images, log into the {{< ver-ref project="os" page="/install/quickstart/aws/host-containers" hash="#interacting-with-a-bottlerocket-node-through-host-containers" >}}admin container{{</ ver-ref >}} and use `ctr`'s `image` commands with the address pointed at `/run/dockershim.sock`. For example, on Kubernetes variants to list all images:
+
+```shell
+ctr --address "/run/dockershim.sock" -n k8s.io image ls
```
Comment (bcressey) [inline content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown:16]: This symlink might not be present in newer k8s variants.

File: content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown (hunk 1 above)
Comment (stockholmux) [reply content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown]: D'oh. Thanks!

File: content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown (hunk 2 above)
Comment (stockholmux) [reply content/en/faqitems/8_1-management-how-do-i-clean-up-images.markdown:16]: Looks like its [still in 1.28](https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/kubernetes-1.28/dockershim-symlink.conf) - is there a planned removal?

**Review by gthao313 — APPROVED** [review]

**Pull request merged**